.splunk-version-hash

```diff
@@ -0,0 +1,2 @@
+a1a6394cc5ae
```

Comment:
Straight copy of https://github.com/openshift/splunk-forwarder-operator/blob/c28e0fb044330d413e18cd7f0a43ab1673d2aa18/.splunk-version-hash